geoapps/inversion/line_sweep/driver.py

```diff
@@ -39,20 +39,23 @@ def run(self):  # pylint: disable=W0221
 
     def setup_params(self):
         h5_file_path = Path(self.workspace.h5file).resolve()
-        ui_json_path = h5_file_path.with_suffix(".ui.json")
-        if not ui_json_path.is_file():
+        ui_json_path = h5_file_path.parent / h5_file_path.stem.replace(
+            ".ui", ".ui.json"
+        )
```

Comment:
**suggestion:** do not remove ".ui" just anywhere in the name but only at the end
```suggestion
        ui_json_path = h5_file_path.parent / (re.sub(r"\.ui$", "", h5_file_path.stem) + ".ui.json")
```